frontend/src/pages/projectView/ProjectView.tsx

```diff
@@ -0,0 +1,75 @@
+import { useParams } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import allTasks from './tasks';
+import projects from './projects';
+import thumbnail from '../../assets/images/Rectangle 3025.svg';
+import './projectView.css';
+import ProjectListView from '../../components/projectListView/ProjectListView';
+
+export type Task = {
+  id: number | string;
+  description: string;
+  status: string;
+  due_date: string;
+  assignee: string;
+  completed: boolean;
+  priority: string;
+  project_id: number | undefined;
+  user_id: number | string;
+};
+
+type Project = {
+  id: number;
+  title: string;
+  description: string;
+  thumbnail_link: string;
+  date_of_creation: string;
+  amount_of_tasks: number;
+  user_id: number;
+};
```

Comment:
I assume these objects are going to be used elsewhere, maybe it is better to extract them out into their own files